widgets/src/Inputs/Viewable.jsx

```diff
@@ -76,7 +78,7 @@ const EditButtonContainer = styled.div`
 
 return (
   <Container className={big ? "big" : ""}>
-    <Row>
+    <Stack direction={flexDirection}>
```

Comment:
We should use the direction as a class instead of a variable to avoid unnecessary rerenders